src/pymor/core/base.py

```diff
@@ -136,7 +136,7 @@ def __auto_init(self, locals_):
                 if arg not in self.__dict__:
                     setattr(self, arg, locals_[arg])
 
-        auto_init_name = f'_{classname}__auto_init'
+        auto_init_name = f"_{classname.lstrip('_')}__auto_init"
```

Comment:
What is going on here with the `lstrip`?!

Reply:
https://github.com/pymor/pymor/pull/2064#issuecomment-1620437188

Reply:
Ah ok, I see.